src/omp_autotune.c

```diff
@@ -143,7 +143,7 @@ void omp_autotune_run(struct db_main *db)
 		printf("\n");
```

Comment:
python
("ahh_<>\$sssh")/:
["hatem"/]()
@hatemm
```suggestion
 		printf("\n");
```
name: CodeQL analysis
@vanessayuenn
Hello git history spelunker!
3 months ago
2

3
on:
4
  push:
@rneatherway
@heiskr
Update codeql.yml (#16791)
22 days ago
5
    branches: main
6
  pull_request:
7
    branches: main
@vanessayuenn
Hello git history spelunker!
3 months ago
8
    paths:
@nschonni
chore: Add Prettier for Yaml formatting
2 months ago
9
      - '**/*.js'
10
      - '.github/workflows/codeql.yml'
@vanessayuenn
Hello git history spelunker!
3 months ago
11

12
jobs:
13
  build:
@nschonni
chore: Add Prettier for Yaml formatting
2 months ago
14
    runs-on: ubuntu-latest
@vanessayuenn
Hello git history spelunker!
3 months ago
15

16
    steps:
@nschonni
chore: Add Prettier for Yaml formatting
2 months ago
17
      - uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f
18
      - uses: github/codeql-action/init@v1
19
        with:
20
          languages: javascript # comma separated list of values from {go, python, javascript, java, cpp, csharp} (not YET ruby, sorry!)
21
      - uses: github/codeql-action/analyze@v1
22
        continue-on-error: true